libraries/AP_NavEKF3/AP_NavEKF3_core.cpp

```diff
@@ -472,15 +472,6 @@ bool NavEKF3_core::InitialiseFilterBootstrap(void)
     // update sensor selection (for affinity)
     update_sensor_selection();
 
-    // If we are a plane and don't have GPS lock then don't initialise
-    if (assume_zero_sideslip() && dal.gps().status(preferred_gps) < AP_DAL_GPS::GPS_OK_FIX_3D) {
```

Comment:
this could be added as an EK3_OPTIONS bit, but not by default.

Reply:
Will give that a try!